src/components/QuranReader/TranslationView/TranslationText/FootnoteText.tsx

```diff
@@ -4,34 +4,43 @@ import React, { MouseEvent } from 'react';
 
 import classNames from 'classnames';
 import useTranslation from 'next-translate/useTranslation';
+import { useRouter } from 'next/router';
 
 import CloseIcon from '../../../../../public/icons/close.svg';
+import {
+  getLanguageDirectionById,
+  getLanguageFontById,
+  findLanguageIdByLocale,
+} from '../../../../utils/locale';
 
 import styles from './FootnoteText.module.scss';
+import transStyles from './TranslationText.module.scss';
 
 import Button, { ButtonSize, ButtonShape, ButtonType } from 'src/components/dls/Button/Button';
 import Spinner from 'src/components/dls/Spinner/Spinner';
-import useDirection from 'src/hooks/useDirection';
+import Footnote from 'types/Footnote';
 
 interface FootnoteTextProps {
-  text: string;
+  footnote: Footnote;
   onCloseClicked: () => void;
   onTextClicked?: (event: MouseEvent, isSubFootnote?: boolean) => void;
   isLoading?: boolean;
-  isStaticContent?: boolean;
 }
 
 const FootnoteText: React.FC<FootnoteTextProps> = ({
-  text,
+  footnote,
   onCloseClicked,
   onTextClicked,
   isLoading,
-  isStaticContent = false,
 }) => {
-  const direction = useDirection();
+  const { locale } = useRouter();
```

Comment:
you can access `lang` inside `useTranslation` instead of using `useRouter` in addition to `useTranslation`